ExtraDry/Sample.Data/SampleContext.cs

```diff
@@ -39,6 +39,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         //modelBuilder.Entity<Content>().Property(e => e.Layout).HasColumnType()
 
+        modelBuilder.Entity<Region>().Property(e => e.Uuid).HasDefaultValueSql("NEWID()");
```

Comment:
Not generally used, we don't want the DB controlling data values, what was the reason for this one?

Reply:
It was to make sure that the migration that was created in the sample project to add a uuid to the DB would generate new ids for each entry. I did it this way because the alternative was to manually alter the generated migration.

Reply:
As this is a sample that runs locally against sample data, I've been 
1. Reverting all migrations
2. Creating a single new migrations
3. Ensure the sample data setup does everything
As there are no backward-compatibility requirements...

Reply:
Nice, a shortcut, I've removed this line and rolled it all into the first migration